src/commands/lambda/__tests__/uninstrument.test.ts

```diff
@@ -583,160 +548,16 @@ describe('lambda', () => {
-              FunctionName: 'lambda-2-us-east-1',
-              Handler: 'index.handler',
-              Runtime: 'nodejs16.x',
-            },
-          },
-          'arn:aws:lambda:us-east-1:123456789012:function:lambda-3-us-east-1': {
-            config: {
-              FunctionArn: 'arn:aws:lambda:us-east-1:123456789012:function:lambda-3-us-east-1',
-              FunctionName: 'lambda-3-us-east-1',
-              Handler: 'index.handler',
-              Runtime: 'nodejs16.x',
-            },
-          },
-          'arn:aws:lambda:us-east-2:123456789012:function:lambda-1-us-east-2': {
-            config: {
-              FunctionArn: 'arn:aws:lambda:us-east-2:123456789012:function:lambda-1-us-east-2',
-              FunctionName: 'lambda-1-us-east-2',
-              Handler: 'index.handler',
-              Runtime: 'nodejs18.x',
-            },
-          },
-          'arn:aws:lambda:us-east-2:123456789012:function:lambda-2-us-east-2': {
-            config: {
-              FunctionArn: 'arn:aws:lambda:us-east-2:123456789012:function:lambda-2-us-east-2',
-              FunctionName: 'lambda-2-us-east-2',
-              Handler: 'index.handler',
-              Runtime: 'nodejs18.x',
-            },
-          },
-          'arn:aws:lambda:us-east-2:123456789012:function:lambda-3-us-east-2': {
-            config: {
-              FunctionArn: 'arn:aws:lambda:us-east-2:123456789012:function:lambda-3-us-east-2',
-              FunctionName: 'lambda-3-us-east-2',
-              Handler: 'index.handler',
-              Runtime: 'nodejs20.x',
-            },
-          },
-        })
-
-        for (const failingLambda of failingLambdas) {
-          lambdaClientMock
-            .on(UpdateFunctionConfigurationCommand, {FunctionName: failingLambda})
-            .rejects('Unexpected error updating request')
-        }
-
-        const cli = makeCli()
-        const context = createMockContext()
-        const code = await cli.run(
-          [
-            'lambda',
-            'instrument',
-            '-f',
-            'arn:aws:lambda:us-east-1:123456789012:function:lambda-1-us-east-1',
-            '-f',
-            'arn:aws:lambda:us-east-1:123456789012:function:lambda-2-us-east-1',
-            '-f',
-            'arn:aws:lambda:us-east-1:123456789012:function:lambda-3-us-east-1',
-            '-f',
-            'arn:aws:lambda:us-east-2:123456789012:function:lambda-1-us-east-2',
-            '-f',
-            'arn:aws:lambda:us-east-2:123456789012:function:lambda-2-us-east-2',
-            '-f',
-            'arn:aws:lambda:us-east-2:123456789012:function:lambda-3-us-east-2',
-          ],
-          context
-        )
-        const output = context.stdout.toString()
-        expect(code).toBe(0)
-        expect(output).toMatchSnapshot()
-      })
-
-      test('aborts when every lambda function fails to update on uninstrument', async () => {
```

Comment:
Also moved this whole test to the [test of the same name](https://github.com/DataDog/datadog-ci/blob/0944aea9bb1f61ad2a05faaf6978a367ba61d6e0/src/commands/lambda/__tests__/instrument.test.ts#L1273) in `src/commands/lambda/__tests__/instrument.test.ts`, because it was using `cli.run()` with the `lambda instrument` command while it was in the `uninstrument.test.ts` file.